plugins/cue_points/base/CuePointPlugin.php

```diff
@@ -289,7 +289,9 @@ public static function getCuePointSearchData(entry $entry)
 			return;
 		
 		$dataByType = array();
+		CuePointPeer::setUseCriteriaFilter(false);
```

Comment:
I took care of it by adding: " $criteria->add(CuePointPeer::STATUS, CuePointStatus::DELETED, Criteria::NOT_EQUAL); " to the retrieveByEntryId method.